spec/views/components/pageflow/admin/add_membership_button_if_needed_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+
+module Pageflow
+  module Admin
+    describe AddMembershipButtonIfNeeded, type: :view_component do
+      before do
+        helper.extend(ActiveAdmin::ViewHelpers)
+        helper.extend(Pageflow::Admin::MembershipsHelper)
+        allow(helper).to receive(:new_admin_account_membership_path)
+        allow(helper).to receive(:membership_possible_for)
```

Comment:
Here as well as with `url_for`, the exception backtrace changed. The individual specs did already pass without these calls, and maybe running the test suite again I will find that having `url_for` stubbed makes the suite pass already